networking/configuring-cluster-network-range.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="configuring-cluster-network-range"]
+= Configuring the cluster network range
+include::_attributes/common-attributes.adoc[]
+:context: configuring-cluster-network-range
+
+toc::[]
+
+As a cluster administrator, you can expand the cluster network range after cluster installation. You might want to expand the cluster network range if you need more IP addresses for additional nodes. For example, if you deployed a cluster an specified `10.128.0.0/19` as the cluster network range, you are limited to 16 nodes. You can expand that to 510 nodes by changing the CIDR mask on a cluster to `/14`.
```

Comment:
there is another aspect to the number of nodes you are limited to. the hostPrefix. The hostPrefix defines how
many ip's you can have on each node. in this example, /19 gives you 8192 total ip addresses. if the hostPrefix is 23 that allows 512 ip addresses per node which gives you the 16 node count you wrote (8192 / 512 = 16)

just pointing this out, because as an example, the hostPrefix could be /21 which allows for 2048 ip's per node. In that case, with a /19 mask the cluster could only have a max of 4 nodes.

hope I'm making sense :)